stdlib/public/SDK/Network/NWPath.swift

```diff
@@ -74,7 +74,8 @@ public struct NWInterface : Hashable, CustomDebugStringConvertible {
 			}
 		}
 	}
-
+	
+	/// The interface types wifi, cellular, wiredEthernet, and loopback.
```

Comment:
This should be singular; i.e., “The interface type.” Again, an enumeration doesn’t need to be enumerated in the documentation.